doc/startup.md

```diff
@@ -111,6 +111,8 @@ For security purposes, it is important to delete the seed account (as the userna
 
 Additionally you should remove the `adminUserDefault` and `adminPassDefault` lines from `config/lrsql.json`. This will prevent the system from re-creating the seed accounts on future restarts.
 
+Note that SQL LRS will not delete an admin account if it is the only one present.
```

Comment:
Should document that this behavior does not occur for OIDC credentials. Maybe not on the startup page but definitely somewhere.

Reply:
I can put it in the OIDC docs

Reply:
https://github.com/yetanalytics/lrsql/pull/294/commits/80aa8061ce935b9c2267a5f30991d335b34152b7

Reply:
Noticed some grammar and spelling errors in your sentence:
> Note that when OIDC and local accounts are enabled SQL LRS will allow the deletion of all local accounts from the aministrative interface and API.

Here's it fixed:
> Note that when OIDC and local accounts are enabled**,** SQL LRS will allow the deletion of all local accounts from the **administrative** interface and API.

Though even then I think it can be worded better.

Reply:
https://github.com/yetanalytics/lrsql/pull/294/commits/efb62fe830189fd3f0d0d3a7b7903421ded86338

I think it's worded fine, this is minor.